LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Alejandro Barreto
```

Comment:
Make this National Instruments, not you.

Reply:
Done!